types/types.proto

```diff
@@ -366,10 +366,20 @@ message ContainerLogsRequest {
   bool    stderr      = 7;
 }
 
+
 message ContainerLogsResponse {
   bytes log = 1;
 }
 
+message WaitRequest{
+    string container = 1;
+    string processId = 2;
+}
```

Comment:
it seems better if there is a `bool noHang = 3`.
it can be added later if it is needed.